src/pages/en/guides/environment-variables.md

```diff
@@ -38,6 +49,7 @@ To achieve, you can create an `env.d.ts` in `src` directory, then augment `Impor
 
 ```ts
 interface ImportMetaEnv {
+  readonly DB_PASSWORD: string;
```

Comment:
maybe a note here like `// Remember: this will only be available when run on the server!`